packages/@react-aria/tooltip/docs/useTooltipTrigger.mdx

```diff
@@ -111,18 +113,62 @@ function Example(props) {
 
   return (
     <span style={{position: 'relative'}}>
-      <button ref={ref} {...triggerProps}>I have a tooltip</button>
+      <button ref={ref} {...triggerProps} style={{fontSize: 18}} onClick={() => alert('Pressed button')}>{props.children}</button>
       {state.isOpen && (
-        <Tooltip state={state} {...tooltipProps}>
-          And the tooltip tells you more information.
-        </Tooltip>
+        <Tooltip state={state} {...tooltipProps}>{props.tooltip}</Tooltip>
       )}
     </span>
   );
 }
 
-<Example />
-<Example />
+<TooltipButton tooltip="Edit">🖊</TooltipButton>
```

Comment:
it's probably always been bad, but these icons are super hard to see in dark mode